tests/b_frames.pbtxt

```diff
@@ -3,13 +3,6 @@ test {
         id: "b_frames"
         description: "B-Frame Testing"
     }
-    input {
-        filepath: "/tmp/akiyo_qcif.yuv"
```

Comment:
I do not see the point. These tests are to be used as templates and for encode_verify.py to run.
And to be used with the data prepare in the prepare script. This just makes things more difficult.

Reply:
I disagree, in particular for all the input keys except filepath: Right now, you will get input.resolution = "176x144" unless you set the value explicitly using "-e input.resolution 1280x720". This is very confusing IMO: The first thing a user will do is testing her own video file.

Note that the revised instructions suggest using a y4m raw file instead of a yuv raw one. If the file is y4m, it will fill up the input dictionary with the values obtained from y4m.

Reply:
OK. Let's update the prepare test script with a y4m and put that into the test. I need the encapp_verify.py to work as a lowest bar on working setup.

Reply:
> OK. Let's update the prepare test script with a y4m and put that into the test. I need the encapp_verify.py to work as a lowest bar on working setup.

Main concern is that it adds 11 Mb to the repo. 

The previous mechanism (where the prepare_* script will do this for you) should work equally